app/models/user.rb

```diff
@@ -181,11 +181,9 @@ def self.authenticate(options)
       transitions :from => :pending, :to => :active
     end
 
-    # Used in test suite, not something that we would normally need to do.
-    if Rails.env.test?
```

Comment:
This is only being removed so that we can deactivate while revalidating an auth id association and as explained above I don't think we can do that so this can then be left as is.